lib/mix/lib/mix/dep/loader.ex

```diff
@@ -304,8 +304,8 @@ defmodule Mix.Dep.Loader do
     |> elem(0)
   end
 
-  defp rebar_children(root_config, manager) do
-    from = Path.absname("rebar.config")
+  defp rebar_children(root_config, manager, dest) do
+    from = Path.join(dest, "rebar.config") |> Path.absname
```

Comment:
I am with @eksperimental. You shall never pipe once, period. If you start with `dest |>`, then this is fine.